evaluation/create_function_usecase.md

```diff
@@ -0,0 +1,70 @@
+## Create A Function
+### Primary Actor: Advanced User
+
+### Trigger
+The user wants to use a specific, not included function to solve a given task.
+
+### Preconditions
+1. The user is logged in.
+
+### Main success scenarion
+1. The user clicks on "Switch to Developer Suite".
+2. The user confirms the warning pop-up to enter the advanced section Developer Suite.
+3. The user switches to the second tab "Manage Functions".
+4. The user switches to the second sub-tab "Create a new Function".
+5. The user unlocks the protection mode to enter the edit mode.
+6. The user specifies general information about the function, information about the result of the function and specifies fields.
+7. The user saves and finishes the Function creation process.
+
+### Postconditions
+1. The Function is added to the database.
+2. APIs can be added for the which allows the user to use the Function in the Routine building process.
```

Comment:
Great, I also added postcondition in each of the 3 API use-cases, even it was oblivious